packages/autocertifier/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "@streamr/autocertifier",
```

Comment:
As we have a separate package for `autocertifier-client`, would it make sense to rename this to `autocertifier-server`?

Reply:
Yeah the autocertifier is acting as a server, it has a DnsServer and a REST server for requesting subdomains and certificates. I would not rename it for now